spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -112,7 +112,6 @@ object DynamicConformanceJob {
 
       log.info("Conformance finished successfully")
     } finally {
-      Atum.getControlMeasure.runUniqueId
```

Comment:
Just curious, what was the point of this call?

Reply:
Maybe leftover code. The problem with this line was that it was hiding the proper exception which was had the message ControlFramework tracking not initialized